### PR TITLE
Fix low-hanging fruit from QA findings

### LINKED ISSUE #1736: "Download the Relay extension for Firefox" is not visible as a hyperlink on homepage
**Build:**
Stage

**Affected platforms:**
Windows 10 64-bit

**Browsers tested on:**
Firefox, latest release

**Prerequisites:**
N/A

**Steps to reproduce:**

1. Open the Relay homepage.
2. Scroll to "How it works" section and observe the text under the "Get the extension" subtitle.
3. Click on the "Download the Relay extension for Firefox" text.

**Actual result:**
1) "Download the Relay extension for Firefox" is not visible as hyperlink.
2) There's no space after dot (before the "Generate email..." text).

**Expected result:**
1) "Download the Relay extension for Firefox" is visible as hyperlink.
2) There's a space after dot (before the "Generate email..." text).

**Notes:**
N/A

link not visible no space after dot

### LINKED ISSUE #1740: New tab icon not displayed anymore for "Manage your Firefox Account" link on Profile menu
**Build:**
Stage

**Affected platforms:**
Windows 10 64-bit

**Browsers tested on:**
Firefox, latest release

**Prerequisites:**
N/A

**Steps to reproduce:**

1. Navigate to Relay page and sign in using an existing Firefox account.
2. Click the Profile button to open the Profile menu.
3. Observe the "Manage your Firefox Account" link.

**Actual result:**
The new tab icon is not displayed anymore but the page opens in a new tab.

**Expected result:**
The new tab icon is displayed. 

**Notes:**
The same issue occurs for Relay Settings links.

**React:**
new tab icon missing

no new tab icon

**Before:**
before
before

### LINKED ISSUE #1742: "Search aliases" bar not displayed anymore for free users
**Build:**
Stage

**Affected platforms:**
Windows 10 64-bit

**Browsers tested on:**
Firefox, latest release

**Prerequisites:**
The user has no active subscription.
The user has a few aliases already created.

**Steps to reproduce:**

1. Navigate to Relay page and sign in.
2. On dashboard go to the aliases list and look for Search aliases bar just above the list.

**Actual result:**
"Search aliases" bar is not displayed anymore for free users no matter if the VPN is set in country where Premium is available or not.

**Expected result:**
"Search aliases" bar is displayed for free users.

**Notes:**
N/A

**React:**
no search bar

**Before:**
search bar before

### PR DESCRIPTION
**Pull request by Vinnl**
(I'll update this PR during the day as I catch more QA findings.)

This PR fixes #1736.

image

This PR fixes #1740.

image

image

This PR fixes #1742.

image

- [ ] l10n changes have been submitted to the l10n repository, if any.
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] I've added or updated relevant docs in the docs/ directory.
- [ ] All UI revisions follow the [coding standards](https://github.com/mozilla/fx-private-relay/blob/main/docs/coding-standards.md), and use Protocol tokens where applicable (see `/static/scss/libs/protocol/css/includes/tokens/dist/index.scss`).
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).

### CODE REVIEW

**Review by groovecoder — APPROVED** [review]
Code and spot-checks LGTM. Would like @maxxcrawford to check the SCSS too.

File: frontend/src/components/layout/UserMenu.tsx [hunk 1]
```diff
@@ -136,6 +137,7 @@ export const UserMenu = () => {
             className={styles["settings-link"]}
           >
             {l10n.getString("nav-profile-manage-fxa")}
+            <NewTabIcon alt="" />
```
[the remark this answers is not in the document]
Comment (Vinnl) [reply frontend/src/components/layout/UserMenu.tsx:140]: This was an edge case to me; it's not necessarily a purely decorative image, but something like "this link opens in a new tab" felt a bit too noise for me. Let me ask the accessibility experts.

[the remark this answers is not in the document]
Comment (Vinnl) [reply frontend/src/components/layout/UserMenu.tsx:140]: Response came in: yes, this one could use content. @codemist do you want to add that?

[the remark this answers is not in the document]
Comment (codemist) [reply frontend/src/components/layout/UserMenu.tsx:140]: Do alt texts need to be localized?

[the remark this answers is not in the document]
Comment (groovecoder) [reply frontend/src/components/layout/UserMenu.tsx:140]: Yeah, I'm sure we want alt text translated too.

**Review by maxxcrawford — APPROVED** [review]
Icon matches Nebula ✅ 
Listed issues are resolved when testing ✅ 
Let's go!!

**Pull request merged**